scripts/reporting/ncanda_quality_control_script.py

```diff
@@ -300,12 +301,11 @@ def main(args):
         for e in error:
     		if e == 'null':
     			error.remove(e)
-        f = csv.writer(open("missing_form.csv", "wb+"))
-    	f.writerow(["subject_site_id", "visit_date", "np_missing", "event_name", "error"])
+        f = csv.writer(open(args.csvdir, "wb+"))
```

Comment:
note that in #55 this is also being modified... will likely cause merge conflicts...